src/Command.js

```diff
@@ -15,10 +16,9 @@ module.exports = BotModule.extend({
     initialize: function (dependencyGraph) {
         this._super(dependencyGraph);
 
-        let self = this;
         this.discordClient.on('message', message => {
             let messageText = message.content;
-            if (messageText.substr(0, 1) !== self.commandPrefix) return;
+            if (messageText.substr(0, 1) !== this.commandPrefix) return;
```

Comment:
Why did you change `self` to `this`?  Isn't `this` in this context dependant on the `.on` method?